packages/edit-site/src/components/global-styles/screen-revisions/revisions-buttons.js

```diff
@@ -25,19 +25,20 @@ function ChangesSummary( { revision, previousRevision } ) {
 	const changes = getGlobalStylesChanges( revision, previousRevision, {
 		maxResults: 7,
 	} );
-	const changesLength = changes.length;
 
-	if ( ! changesLength ) {
+	if ( ! changes.length ) {
 		return null;
 	}
 
 	return (
-		<span
+		<ul
```

Comment:
Similar to [this comment](https://github.com/WordPress/gutenberg/pull/59055/files#r1492310046), I think it would be better to add `role="list"`.